src/constants/app.const.ts

```diff
@@ -6,3 +6,4 @@ export const COMPANY_NOT_ACTIVE_ERROR_MSG = 'The company is not currently active
 export const COMPANY_OVERSEAS_ERROR_MSG = 'This is an overseas company, and cannot be closed using this service.'
 export const NO_ACTIVE_DIRECTORS_ERROR_MSG = 'The company has no active directors.'
 export const NO_ACTIVE_MEMBERS_ERROR_MSG = 'The company has no active members.'
+export const FEEDBACK_LINK = 'https://www.research.net/r/closing-a-company-feedback'
```

Comment:
ACs specify two different addresses. We are using one in both places here. Is this correct?

Reply:
Ahh you are right, ill get that fixed